oracle-plugin/src/e2e-test/java/io.cdap.plugin/oracle/BQValidation.java

```diff
@@ -1,129 +1,146 @@
-/*
- * Copyright © 2023 Cask Data, Inc.
- *
- * Licensed under the Apache License, Version 2.0 (the "License"); you may not
- * use this file except in compliance with the License. You may obtain a copy of
- * the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
- * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
- * License for the specific language governing permissions and limitations under
- * the License.
- */
-
 package io.cdap.plugin.oracle;
 
 import com.google.cloud.bigquery.TableResult;
-import com.google.common.collect.MapDifference;
-import com.google.common.collect.Maps;
 import com.google.gson.Gson;
-import com.google.gson.JsonArray;
 import com.google.gson.JsonObject;
-import com.google.gson.reflect.TypeToken;
 import io.cdap.e2e.utils.BigQueryClient;
 import io.cdap.e2e.utils.PluginPropertyUtils;
 import io.cdap.plugin.OracleClient;
+import org.junit.Assert;
 
 import java.io.IOException;
-import java.lang.reflect.Type;
 import java.sql.Connection;
 import java.sql.ResultSet;
+import java.sql.ResultSetMetaData;
 import java.sql.SQLException;
 import java.sql.Statement;
+import java.sql.Timestamp;
+import java.sql.Types;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
 import java.util.ArrayList;
+import java.util.Base64;
+import java.util.Date;
 import java.util.List;
-import java.util.Map;
 
-/**
- *  Big Query client
- */
 public class BQValidation {
 
-  /**
-   * Extracts entire data from source and target tables.
-   * @param sourceTable table at the source side
-   * @param targetTable table at the BigQuery side
-   * @return true if the values in source and target side are equal
-   */
-  public static void validateBQAndDBRecordValues(String schema, String sourceTable, String targetTable)
-    throws SQLException, ClassNotFoundException, IOException, InterruptedException {
+  public static boolean validateBQAndDBRecordValues(String schema, String sourceTable, String targetTable)
+    throws SQLException, ClassNotFoundException, ParseException, IOException, InterruptedException {
+    List<JsonObject> jsonObject = new ArrayList<>();
```

Comment:
Change the name of this list from jsonObject to something relevant. This contains the Response in Json so something related to that.

Reply:
Changed it to jsonResponse.